website/static/js/project-organizer.js

```diff
@@ -364,10 +375,14 @@ var tbOptions = {
                     }
                     tb.filter(event);
                 },
-                onchange: m.withAttr('value', tb.filterText),
+                onchange: function() {
+                    m.withAttr('value', tb.filterText);
```

Comment:
Yes, I was thinking the same thing, search doesn't seem to be affected if this line is present or not.